cassandra/cqltypes.py

```diff
@@ -1026,7 +1026,7 @@ def serialize_safe(cls, val, protocol_version):
             try:
                 item = val[i]
             except TypeError:
-                item = getattr(val, fieldname)
+                item = getattr(val, fieldname, None)
```

Comment:
I would add a warning here, so user would notice that their object doesn't match the scheme.
cause in some cases they would do that on purpose, but in some case they would want to know about it and align their code with the actual scheme.

Reply:
So @fruch should I include a warning?

Reply:
There is a better flow to recreate the UDT in case some field needs to be deprecated

I'm more worried about the case that the client code isn't aware of new fields.

So yes please add a warning, and a nice touch to it would be an environment variable to disable, like cqlengine is doing in other places.

Also I would recommend suggesting it to the upstream cassandra-driver, this case is not unique to Scylla, and we take fixes from upstream on a regular basis